src/controller/CHIPDeviceControllerSystemState.h

```diff
@@ -81,8 +82,10 @@ struct DeviceControllerSystemStateParams
 #if CONFIG_NETWORK_LAYER_BLE
     Ble::BleLayer * bleLayer = nullptr;
 #endif
-    Credentials::GroupDataProvider * groupDataProvider = nullptr;
-    Crypto::SessionKeystore * sessionKeystore          = nullptr;
+    Credentials::GroupDataProvider * groupDataProvider                   = nullptr;
+    chip::app::reporting::ReportScheduler::TimerDelegate * timerDelegate = nullptr;
+    chip::app::reporting::ReportScheduler * reportScheduler              = nullptr;
```

Comment:
These should not be here.  They should be in the "Params that will be deallocated via Platform::Delete" section below, no?